Explorer/Assets/StreamingAssets/Js/Modules/EthereumController.js

```diff
@@ -1,4 +1,4 @@
 module.exports.sendAsync = async function(message) {
-    console.log('JSMODULE: sendAsync')
-    return {};
+    const result = await UnityEngineApi.SendEthereumMessageAsync(message.id, message.method, JSON.stringify(message.params))
```

Comment:
I think `message.params` are already in JSON according to https://github.com/decentraland/js-sdk-toolchain/blob/456ae24367274c59900cacbaa48ce8c380da69f9/packages/%40dcl/sdk/src/internal/provider.ts. Have you tried to marshal it directly as a string?

```
export function getEthereumProvider(sendAsync: SendAsyncType) {
  async function request(message: RPCSendableMessage) {
    const response = await sendAsync({
      id: message.id,
      method: message.method,
      jsonParams: JSON.stringify(message.params)
    })
    return JSON.parse(response.jsonAnyResponse)
  }
```